features/0434-outofband/README.md

```diff
@@ -0,0 +1,417 @@
+# Aries RFC 0434: Out of Band Protocol 1.0
+
+- Authors: [Ryan West](ryan.west@sovrin.org), [Daniel Bluhm](daniel.bluhm@sovrin.org), Matthew Hailstone, [Stephen Curran](swcurran@cloudcompass.ca), [Sam Curren](sam@sovrin.org), [George Aristy](george.aristy@securekey.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-03-01
+- Status Note: This RFC extracts the `invitation` messages from the [DID Exchange](../../features/0023-did-exchange/README.md) protocol (and perhaps [Connection](../../features/0160-connection-protocol/README.md)), and replaces the combined `present_proof/1.0/request` combined with the `~service` decorator to define an ephemeral (connection-less) challenge.
```

Comment:
nit: this note looks like it belongs in the summary section

Reply:
For transition things, I like to leave them out of the main narrative since eventually this is "normal" and new people reading it don't to know how things used to be. That opinion is a nit as well :-)

Willing to change, but that's the reason I put it there.